plugins/Toolbox/resources/qml/ToolboxDownloadsShowcaseTile.qml

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) 2018 Ultimaker B.V.
 // Toolbox is released under the terms of the LGPLv3 or higher.
 
-import QtQuick 2.2
+import QtQuick 2.3
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import UM 1.1 as UM
 
 Item
 {
     width: UM.Theme.getSize("toolbox_thumbnail_large").width
-    height: childrenRect.height
+    height: thumbnail.height + packageName.height
```

Comment:
Out of curiosity, why change this?

Reply:
childrenRect is a nice concept, but it leads to binding errors when doing too many creative anchorings (ie: a child anchor to the parent when the parent size depends on the positioning of the children).

Reply:
childrenRect is safe to use for eg columns and rows because those don't rely on anchoring.

Reply:
Good to know.

Reply:
Unless your column/row items have a variable/collapsible height :P